common/openapi.yml

```diff
@@ -503,6 +522,10 @@ components:
             linkExpireTime:
               type: string
               format: date-time
+    GetAuthUrlRequest:
+      type: object
+      additionalProperties:
+        type: string
```

Comment:
nit, and optional: provide an example here to make the swagger-ui a little prettier, like
```
    GetAuthUrlRequest:
      type: object
      additionalProperties:
        type: string
      example: |
        {"redirectTo": "https://my.redirect.url/"}
```